userMap.json

```diff
@@ -26,6 +26,7 @@
     "xeonacid": ["Xeonacid"],
     "xyenchi": ["XYenChi"],
     "yining": ["yining765"],
-    "zenithal": ["ZenithalHourlyRate"]
+    "zenithal": ["ZenithalHourlyRate"],
+    "kxxt": ["kxxt"]
```

Comment:
Please keep the list in alphabetical order, thx.

Reply:
My fault. It has been fixed now.